src/_gettsim_tests/test_naming_conventions.py

```diff
@@ -34,8 +34,8 @@ def time_indep_function_names(all_functions: dict[str, callable]) -> list[str]:
     for function_name, function in all_functions.items():
         if hasattr(function, "__info__"):
             info = function.__info__
-            if "dates_active_dag_key" in info:
-                time_indep_function_names.add(info["dates_active_dag_key"])
+            if "name_in_dag" in info:
```

Comment:
This check should not be necessary anymore?

Reply:
Assuming that users don't manually add `__info__` but use the decorator, then no. Seems reasonable to me, since `__info__` should not become part of the public interface.